test/kixi/stats/core_test.cljc

```diff
@@ -441,26 +464,33 @@
   test-opts
   ;; Take maps like {}, {:x 1}, {:x 2 :y 3} and compute correlation matrix
   (for-all [coll (gen/vector (gen/map (gen/elements [:x :y :z]) gen/int))]
-    (is (=ish (transduce identity (kixi/covariance-matrix {:x :x :y :y :z :z}) coll)
-              (covariance-matrix' coll)))))
+           (is (=ish (transduce identity (kixi/covariance-matrix {:x :x :y :y :z :z}) coll)
```

Comment:
Whitespace. I think I must have updated my .emacs.d to set the indentation for `for-all`. I reckon it's better without the additional space (+ occurrences below)